dev/io.openliberty.org.eclipse.microprofile/openapi.4.1.bnd

```diff
@@ -19,9 +19,9 @@ Export-Package: \
   org.eclipse.microprofile.openapi.*
 
 Include-Resource: \
-  @${repo;org.eclipse.microprofile.openapi:microprofile-openapi-api;[4.0,4.1)}
+  @${repo;org.eclipse.microprofile.openapi:microprofile-openapi-api;[4.1,4.2)}
 
 -maven-dependencies: \
-   dep1;groupId=org.eclipse.microprofile.openapi;artifactId=microprofile-openapi-api;version=4.0.2;scope=runtime
+   dep1;groupId=org.eclipse.microprofile.openapi;artifactId=microprofile-openapi-api;version=4.1;scope=runtime
```

Comment:
Does this need to be precise i.e. 4.1-RC1 for the scanning function to find the resource?

Reply:
Possibly? I'm not sure what this is actually used for...

Reply:
Changed it in case it is important.